### ISSUE TITLE
Support for server-side scripting

### ISSUE THREAD

**Issue by andrewcurioso** [AntonTerekhov/OrientDB-PHP#37]
Currently there appears to be no way to execute server-side script (OrientDB 1.0) via this wrapper.

More documentation is here:
https://github.com/orientechnologies/orientdb/wiki/Javascript-Command

**Comment by lvca** [comment]
OrientDB 1.0 is pretty old, you've to use a more recent version.

**Comment by AntonTerekhov** [comment]
Yes, you're right: no server-side scripting is possible right now.

**Comment by andrewcurioso** [comment]
@lvca I should have specified "since" 1.0. I'm not actually using 1.0.

I started on a patch but I figured it might be faster for someone core to the project to do it. I'd be more than happy to finish the patch / do a push request if it is "low priority" and it will be a while before someone gets to it.

I'm going off of the documentation here:
https://github.com/orientechnologies/orientdb/wiki/Network-Binary-Protocol#wiki-SQL_Script_command_payload
